movies-manager/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "movies-manager",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start:dev": "cross-env NODE_ENV=development webpack-dev-server",
+    "build:dev": "cross-env NODE_ENV=development webpack",
+    "start:prod": "cross-env NODE_ENV=production webpack-dev-server",
+    "build:prod": "cross-env NODE_ENV=production webpack",
+    "stat": "webpack --json --profile >stats.json",
+    "lint": "tslint --config ./tslint.json --project ./tsconfig.json",
```

Comment:
lint is not working

Reply:
was discussed in teams, that lint will be added in Task-5, here just stub

Reply:
ok